### PR TITLE
ISSD-1350 [CP 2.0]  Add alert with help article on disabled keys

### PR DESCRIPTION
**Pull request by pedroliveirav**
[ISSD-94](https://liferay.atlassian.net/browse/ISSD-94)

### CODE REVIEW

**Comment by pedroliveirav** [comment]
ci:test:sf

**Comment by pedroliveirav** [comment]
ci:test:sf

**Comment by pedroliveirav** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json [hunk 1]
```diff
@@ -434,6 +437,7 @@
 	"tcat-portal-non production": "TCAT Portal Non-Production",
 	"tcat-portal-production": "TCAT Portal Production",
 	"team-member": "Team Member",
+	"team-member-x": "Team Member:",
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json:440]: can we move the colon out of the language key and into the html instead, bchan usually prefers it that way

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.jsx [hunk 2]
```diff
@@ -214,25 +249,92 @@ const TeamMembersTable = ({
...
 					</p>
+
+					{!!isSingleSubscribedUser.length && (
+						<div className="align-items-center d-flex pt-3">
+							<ClayCheckbox
+								checked={checkedBoxSubscription}
+								onChange={() =>
+									setCheckedBoxSubscription(
+										(checkedBoxSubcription) =>
+											!checkedBoxSubcription
```
Comment (kylembischof) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/TeamMembers/components/TeamMembersTable/TeamMembersTable.jsx:309]: `checkedBoxSubcription` is spelled incorrectly. it should be `checkedBoxSubscription`. notice the missing "s" in Subscription

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json (hunk 1 above)
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json]: Fixed, thank you!

**Comment by pedroliveirav** [comment]
ci:test:relevant

**Review by kevenleone — APPROVED** [review]
LGTM 🔥 

Good job @pedroliveirav

**Comment by kevenleone** [comment]
ci:forward:force

**Review by samuelleolopes — APPROVED** [review]
Good Job! :fire:

**Comment by pedroliveirav** [comment]
resent: https://github.com/is-solutions-delivery/liferay-portal/pull/4990

**Pull request closed without merging**